.github/ISSUE_TEMPLATE/build-break-infrastructure-issue-template.yml

```
@@ -1,6 +1,6 @@
-name: Report Infrastructure Issue - Build Break
-description: File an infrastructure issue causing build breaks to be investigated by the engineering services team.
-labels: ["First Responder", "Detected By - Customer", "Known Build Error"]
+name: Report an infrastructure issue
+description: Report an infrastructure issue causing build breaks across multiple builds
```

[nit] the wording seems a bit redundant, but I'm not sure I even like this suggestion
```suggestion
description: Report an infrastructure issue causing failures across multiple builds
```